### PR TITLE
Channels Toolkit for Storefront Updates

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2285](https://jira.bigcommerce.com/browse/DEVDOCS-2285)

## What changed?
* Updated screenshots
* Removed some fluff in Building Channel Apps
* Edited content for clarity and brevity in a few spots

### CODE REVIEW

File: docs/api-docs/channels/channels-extending-existing.md [hunk 1]
```diff
@@ -3,19 +3,20 @@
 <div class="otp" id="no-index">
 
 ### On this Page
```
Comment (uwikaiddi) [inline docs/api-docs/channels/channels-extending-existing.md:5]: Remove the phrase "Step #:" from the titles, and adjust the capitalization of the headers accordingly.

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

File: docs/api-docs/channels/channels-extending-existing.md (hunk 1 above)
Comment (aglensmith) [reply docs/api-docs/channels/channels-extending-existing.md:5]: Those were already in the article and not a part of this update, so shouldn't hold up the pull request.

Comment (uwikaiddi) [reply docs/api-docs/channels/channels-extending-existing.md:5]: This correction is in line with the description of the changes you made throughout the document (i.e. "applied style guide updates").

Comment (aglensmith) [reply docs/api-docs/channels/channels-extending-existing.md:5]: Okay. I removed that part.

Comment (aglensmith) [reply docs/api-docs/channels/channels-extending-existing.md:5]: Will address style guide updates in [new ticket](https://jira.bigcommerce.com/browse/DEVDOCS-2360).

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**